composer.json

```diff
@@ -43,7 +43,9 @@
         "doctrine/dbal": "~2.2",
         "swiftmailer/swiftmailer": "~5",
         "monolog/monolog": "^1.4.1",
-        "phpunit/phpunit": "^5 || ^6 || ^7 || ^8"
+        "phpunit/phpunit": "^5 || ^6 || ^7 || ^8",
+        "sllh/composer-versions-check": "^2.0",
+        "rector/rector": "^0.5.21"
```

Comment:
do we need `rector` as a dependency for the release?

Reply:
we must not :) this is sure only temporary.